transaction-volume-anomaly-detection/README.md

```diff
@@ -0,0 +1,55 @@
+# Transaction Volume Anomaly Detection
+
+## Description
+
+This bot detects Transactions with Anomalies in Volume
+
+## Supported Chains
+
+- Ethereum
+- Binance Smart Chain
+- Polygon
+- Optimism
+- Arbitrum
+
+## Alerts
+
+- SUCCESSFUL-INTERNAL-TRANSACTION-VOL-INCREASE
+
+  - Fired when there is unusually high number of successful internal transactions
+  - Severity is always set to "low"
+  - Type is always set to "suspicious"
+  - Metadata fields:
+    - COUNT (Current count of successful transaction)
+    - EXPECTED_BASELINE (Expected baseline count of successful transaction)
+
+- SUCCESSFUL-TRANSACTION-VOL-INCREASE
+
+  - Fired when there is unusually high number of successful transactions
+  - Severity is always set to "low"
+  - Type is always set to "suspicious"
+  - Metadata fields:
+    - COUNT (Current count of successful transaction)
+    - EXPECTED_BASELINE (Expected baseline count of successful transaction)
+
+- FAILED-TRANSACTION-VOL-INCREASE
+
+  - Fired when there is unusually high number of failed transactions
+  - Severity is always set to "high"
+  - Type is always set to "exploit"
+  - Metadata fields:
+    - COUNT (Current count of successful transaction)
+    - EXPECTED_BASELINE (Expected baseline count of successful transaction)
+
+- FAILED-INTERNAL-TRANSACTION-VOL-INCREASE
+
+  - Fired when there is unusually high number of failed internal transactions
+  - Severity is always set to "medium"
+  - Type is always set to "suspicious"
+  - Metadata fields:
+    - COUNT (Current count of successful transaction)
+    - EXPECTED_BASELINE (Expected baseline count of successful transaction)
+
+## Test Data
+
+The agent behaviour can be verified with supplied unit tests
```

Comment:
itd be great to also include a block range that shows an attack it triggered on.

Reply:
The block range for any known hack is around 100k blocks, which will take an insane amount of time to handle, issue one is that sometimes when using a rpc provider the request fails and you need to start from scratch, which makes testing with a range pretty much impossible.